### ISSUE TITLE
Error with certutil when running trust|untrust-dev-cert

### ISSUE THREAD

**Issue by wcarson** [SharePoint/sp-dev-docs#316]
#### Category
- [ ] Question
- [ ] Typo
- [x] Bug
- [ ] Enhancement / Suggestion
- [ ] Additional article idea

#### Expected or Desired Behavior
gulp trust|untrust tasks complete successfully or provide meaningful error messages.

#### Observed Behavior
Tasks fail with "cannot call toString on null" errors in gulp-core-build-serve/certificates.js in the `tryTrustCertificate `& `untrustCertificate` methods.

#### Steps to Reproduce
This looks to be caused by the way the ensureCertUtilExePath method in certificates.js finds certutil. In my case, I have two certutil.exe on my system (one 32bit in System32 & one 64 bit in SysWow64). When "where" is run to find certutil, it returns something like:

```
C:\Windows\System32\certutil.exe
C:\Windows\SysWow64\certutil.exe
```

Since the code is expecting only one entry be found, it tries to spawn certutil using this result as the path, which is obviously invalid so that call fails which eventually triggers the error when the code tries to use certutil. 

This should probably be updated to handle multiple certutil's on a system, although I don't know how much of an edge case that would be. I'm not even sure why there's 2 on my system.

**Comment by patmill** [comment]
Thanks for the bug.

**Comment by iclanton** [comment]
This has been fixed in gulp-core-build-serve 2.1.3, which will be used in the next drop of SPFx and the tools. Thanks for pointing this out!
